packages/ng/src/plugins/ngMaterial/_general.scss

```diff
@@ -0,0 +1,13 @@
+@import
+	'~@angular/material/theming', // Basic file for custom theme
```

Comment:
is this mandatory ?

Reply:
it gives access to the `angular-material-typography` mixin which allow us to easily override fonts all over material components